File: FileShare/ViewModels/Files/FileFormData.cs
using FileShare.Dto;

namespace FileShare.ViewModels.Files;

public class FileFormData
{
    public int? Id { get; set; }
    public string? Name { get; set; }
    public string? Comment { get; set; }
    public string? Extension { get; set; }
    public Size? MinSize { get; set; }
    public int? MaxFileSize { get; set; }
    public string? AllowedExtensions { get; set; }
    
}

File: FileShare/client-app/src/common/interfaces/view-models/file/file-form-data.ts
import type { Size } from "../../common/size";

export interface FileFormData {
  id: number | null;
  name: string | null;
  comment: string | null;
  extension: string | null;
  minSize: Size | null;
  maxFileSize: number | null;
  allowedExtensions: string | null;
}